tasks/build.py

```diff
@@ -52,16 +48,21 @@ def build_easystack_from_pr(pr, event_info):
     log("https_proxy '%s'" % https_proxy)
     load_modules = buildenv.get('load_modules') or ''
     log("load_modules '%s'" % load_modules)
+    return jobs_base_dir, local_tmp, build_job_script, submit_command, slurm_params, cvmfs_customizations, http_proxy, https_proxy, load_modules
+    
 
-
-    # [architecturetargets]
+def get_architecturetargets():
     architecturetargets = config.get_section('architecturetargets')
     arch_target_map = json.loads(architecturetargets.get('arch_target_map'))
     log("arch target map '%s'" % json.dumps(arch_target_map))
+    return arch_target_map
+
 
+def create_directory(pr, jobs_base_dir, event_info):
```

Comment:
Please also add a docstring, maybe also add [type hints](https://docs.python.org/3/library/typing.html) in the function signature:

```python
def create_directory(pr: int, jobs_base_dir: str, event_info: dict) -> (str, str, str)
    """
    Create directory for PR

    :param pr: PR number
    :param jobs_base_dir: location ...
    :returns: tuple of 3 elements: string with datestamp (<year>.<month>)
    """